package/libs/elfutils/Makefile

```diff
@@ -21,7 +21,7 @@ PKG_CPE_ID:=cpe:/a:elfutils_project:elfutils
 
 PKG_FIXUP:=autoreconf
 PKG_INSTALL:=1
-PKG_USE_MIPS16:=0
+PKG_USE_MIPS16:=1
```

Comment:
Just delete PKG_USE_MIPS16:=0, don't need to set it to 1.

Reply:
Yeah, I know it's the default, I just wanted to be explicit for reviewers. :-) I'll let @nbd168 update it as preferred.